Makefile

```diff
@@ -0,0 +1,16 @@
+##
+##usage :
+##-------
+
+build: ## build
+	docker build --tag=evanpurkhiser/tune-manager:latest .
+
+dev: build ## dev
+	docker run -it -p 8080:80 -v library:/library -v staging:/staging -v storage:/storage evanpurkhiser/tune-manager
+
+# DEFAULT
+.DEFAULT_GOAL := help
+help:
+	@grep -E '(^[a-zA-Z_-]+:.*?##.*$$)|(^##)' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}' | sed -e 's/\[32m##/[33m/'
```

Comment:
What does this do?

Reply:
This trick allows a nice looking help when you tip `make`

actually this render

```
usage :                      
-------                      
build                          build
dev                            dev
```

It's compatible withs comments into the `Makefile`. I can delete then if you don't want it.